library-backend/library-api/src/main/java/org/internship/library/api/Book.java

```diff
@@ -1,69 +1,35 @@
 package org.internship.library.api;
 
 /**
- * Book class
+ * This interface defines the behaviour of the BookEntity
```

Comment:
More like 'This interface defines the structure of a Book'. Let's not tie it to the Entity yet.